### DISCUSSION TITLE
Proxy setup shows wrong IP from clients in the web console

### DISCUSSION THREAD

**Discussion by sbrkx** [rustdesk/rustdesk-server-pro#306] [General]
We have successfully setup a proxied setup for rustdesk.
One last issue we can not fix by ourselves: the web interface shows the internal proxy server ip for all clients.
This is very bad for us.
Rustdesk must support analyzing the X-Forwarded-For header in a proxy scenario.
Any chance that this gets fixed?

**Comment by rustdesk** [comment]
On the server side (from proxy -> server), it is raw tcp, not http (from client -> proxy), there is no way for the server to see X-Forwarded-For

**Comment by sbrkx** [comment]
okay not good.
What about adding support for the proxy protocol?
https://www.haproxy.com/blog/use-the-proxy-protocol-to-preserve-a-clients-ip-address
